implementation/jobberknoll/infra/account-repo/postgres-account-repo.ts

```diff
@@ -64,20 +24,22 @@ export class PostgresAccountRepo implements AccountRepo {
     const pool = new Pool(connectionString, POOL_SIZE, true);
 
     using client = await pool.connect();
+    await client.queryObject`
+      DO $$ BEGIN
+        CREATE TYPE ACCOUNT_TYPE AS ENUM ('admin', 'driver', 'inspector', 'passenger');
```

Comment:
There is no `CREATE TYPE IF NOT EXISTS` so we have to effectively do `try { create_type() } catch {}` :c

Reply:
Thanks for the idea, let's write it down and move on.